penelope_aerospace_pl_msgs/action/DvmInspect.action

```diff
@@ -57,6 +57,15 @@ float32[] time_to_completion
 # Refer to penelope_aerospace_pl_msgs/msg/DvmState for the definition
 DvmState[] measurement_state
 
+# Triggers state
+TriggersState triggers_state
+
+# speed between movements
```

Comment:
No live velocity feedback is coming from the Gantrry, so this suggestion is just to clarify this is a requested value and not the actual live value. 
```suggestion
# requested speed for the current movement
```

Reply:
ok